### PR TITLE
Bugfix: Github Actions.

### PR DESCRIPTION
**Pull request by guacke**
**Short description**
This PR is aimed at resolving the current Github Actions failures.

**Related issues**

**Additional context**

**Don't forget:** 
- [x] if new files are created: update the Doxygen file, the collective gqcp.hpp header and the CMake files
- [x] update the documentation on [our website](https://github.com/GQCG/gqcg.github.io) and check if new Python bindings should be added

### CODE REVIEW

File: .github/workflows/gqcp.yml [hunk 1]
```diff
@@ -39,22 +26,21 @@ jobs:
...
-      # Note that we install Libcint in ${CONDA_HOME}
       run: |
         cd /tmp
         git clone -b develop https://github.com/GQCG/libcint.git
         cd libcint
         mkdir build && cd build
-        ${CONDA_HOME}/bin/cmake .. -DUSE_MKL=TRUE -DCMAKE_INSTALL_PREFIX=${CONDA_HOME} -DCMAKE_PREFIX_PATH=${CONDA_HOME}
-        make -j3 && sudo make install
+        cmake .. -DUSE_MKL=TRUE -DCMAKE_INSTALL_PREFIX=$HOME/miniconda -DCMAKE_PREFIX_PATH=$HOME/miniconda
+        make -j2 && sudo make install
```
Comment (lelemmen) [inline .github/workflows/gqcp.yml]:
```suggestion
        make -j 1 && sudo make install
```

File: .github/workflows/gqcp.yml [hunk 2]
```diff
@@ -77,5 +63,5 @@ jobs:
         git clone https://github.com/GQCG/gqcp-link.git
         cd gqcp-link
         mkdir build && cd build
-        ${CONDA_HOME}/bin/cmake .. -DCMAKE_PREFIX_PATH=${CONDA_HOME} -DGQCP_INSTALL_PREFIX=~/.local -Dgqcp_DIR=/home/runner/.local/cmake
-        make -j3 && ./test_driver
+        cmake .. -DCMAKE_PREFIX_PATH=$HOME/miniconda -DGQCP_INSTALL_PREFIX=~/.local -Dgqcp_DIR=/home/runner/.local/cmake
+        make -j2 && ./test_driver
```
Comment (lelemmen) [inline .github/workflows/gqcp.yml]:
```suggestion
        make -j 1 && ./test_driver
```

File: .github/workflows/gqcp.yml [hunk 3]
```diff
@@ -39,22 +26,21 @@ jobs:
...
     # Build & test the library 
     - name: Build GQCP and run tests
       run: |
-        export LIBINT_DATA_PATH=${CONDA_HOME}/share/libint/2.3.1/basis
+        export LIBINT_DATA_PATH=$HOME/miniconda/share/libint/2.3.1/basis
         mkdir build && cd build
-        ${CONDA_HOME}/bin/cmake .. -DBUILD_TESTS=TRUE -DBUILD_BENCHMARKS=TRUE -DBUILD_PYTHON_BINDINGS=TRUE -DCMAKE_INSTALL_PREFIX=~/.local -DCMAKE_PREFIX_PATH=${CONDA_HOME}
-        make -j10 && make test && sudo make install
+        cmake .. -DBUILD_TESTS=TRUE -DBUILD_BENCHMARKS=TRUE -DBUILD_PYTHON_BINDINGS=TRUE -DCMAKE_INSTALL_PREFIX=~/.local -DCMAKE_PREFIX_PATH=$HOME/miniconda
+        make -j2 && env CTEST_OUTPUT_ON_FAILURE=1 make test && sudo make install
```
Comment (lelemmen) [inline .github/workflows/gqcp.yml]:
```suggestion
        make -j 1 && env CTEST_OUTPUT_ON_FAILURE=1 make test && sudo make install
```

**Comment by guacke** [comment]
I agree that having two threads can render the compilation outputs out of sync; however, every output is always matched with the corresponding compilation step. Furthermore, switching to single core could double the build time. As such, I would keep the number of cores requested at the max offered by Github Actions.

**Pull request merged**